src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/NavigationTests.java

```diff
@@ -40,26 +39,37 @@ private void init() {
   }
 
   @Test(groups = "MercuryCuratedNavigationTest_001")
-  public void MercuryCuratedNavigationTest_001_navigateThroughSection() {
+  public void MercuryCuratedNavigationTest_001_navigateThroughCategory() {
     init();
 
     navigate.toPage(MercurySubpages.CC_MAIN_PAGE);
 
-    curatedContent.clickOnCuratedContentElementByIndex(0);
+    curatedContent.clickOnCuratedContentElementByIndex(1);
     loading.handleAsyncPageReload();
 
+    UrlChecker.isPathContainedInCurrentUrl(driver, "/wiki/Category:");
+  }
+
+  @Test(groups = "MercuryCuratedNavigationTest_002")
+  public void MercuryCuratedNavigationTest_002_navigateThroughSection() {
+    init();
+
+    navigate.toPage(MercurySubpages.CC_MAIN_PAGE);
+
+    curatedContent.clickOnCuratedContentElementByIndex(0);
+
     curatedContent
         .isTitleVisible()
         .isLinkToMainPageVisible()
         .isSectionVisible()
         .isCuratedContentItemVisibleByIndex(1);
 
     UrlChecker.isPathContainedInCurrentUrl(
-        driver, MercuryPaths.ROOT_PATH_SECTION + curatedContent.getTitle());
+        driver, MercurySubpages.CC_MAIN_PAGE);
   }
 
-  @Test(groups = "MercuryCuratedNavigationTest_002")
-  public void MercuryCuratedNavigationTest_002_redirectToExistingArticle() {
+  @Test(groups = "MercuryCuratedNavigationTest_003")
+  public void MercuryCuratedNavigationTest_003_redirectToExistingArticle() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)